frontend/src/layout/show/ShowView.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
-import { useShowContext } from 'react-admin';
+import { useResourceContext, useShowContext } from 'react-admin';
 import { Box } from '@mui/material';
 import { useCheckPermissions } from '@semapps/auth-provider';
+import { useDataModel } from '@semapps/semantic-data-provider';
 import BaseView from "../BaseView";
 
 const ShowView = ({ title, actions, children }) => {
   const showContext = useShowContext();
   useCheckPermissions(showContext?.record?.id, 'show');
+
+  const resource = useResourceContext({});
+  const dataModel = useDataModel(resource);
+
+  const recordTitle = showContext?.record?.[dataModel?.fieldsMapping?.title] || '';
```

Comment:
Maybe one day we can use the new `recordRepresentation` for easier handling.

Reply:
Good point! 
I added this commit c5b7281f84dcf3be7f6d55469455da0cb150e6e2 to use recordRepresentation on resources